### ISSUE TITLE
LibNotMR gives different results from libmr.MR

### ISSUE THREAD

**Issue by nikita-petrashen** [ashafaei/OD-test#4]
![image](https://user-images.githubusercontent.com/49820557/139066860-4410f288-2041-4aa7-ba43-004a56eecc0d.png)

**Comment by nikita-petrashen** [comment]
code: 
`import libmr
from matplotlib import pyplot as plt

np.random.seed(2)

taillen = 20
not_mr = LibNotMR(taillen)
mr = libmr.MR()

tail = np.random.randn(100)
test = np.arange(-5, 10, 0.01)

not_mr.fit_high(torch.tensor(tail))

mr.fit_high(tail, taillen)

plt.plot(test, mr.w_score_vector(test), label='libmr')
plt.plot(test, not_mr.w_score(test), label='scipy')
plt.legend()
plt.show()`

**Comment by nikita-petrashen** [comment]
I don't know what is the origin of this, but actually LibNotMR plots look more reasonable
